src/meta/src/controller/mod.rs

```diff
@@ -91,8 +91,21 @@ impl From<ObjectModel<schema::Model>> for PbSchema {
         Self {
             id: value.0.schema_id as _,
             name: value.0.name,
-            database_id: value.0.database_id as _,
+            database_id: value.1.database_id.unwrap() as _,
             owner: value.1.owner_id as _,
         }
     }
 }
+
+impl From<ObjectModel<connection::Model>> for PbConnection {
+    fn from(value: ObjectModel<connection::Model>) -> Self {
+        Self {
+            id: value.1.oid as _,
```

Comment:
Can we remove these "as _" now? As type of model field should have matched type of pb field.

Reply:
Oops, forgot to change this one. I will fix it in the following PR.